src/main/java/duke/task/TaskList.java

```diff
@@ -0,0 +1,96 @@
+package duke.task;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Represents a list of tasks.
+ * This class provides methods to manage tasks in a list, including adding, removing, and retrieving tasks.
+ */
+public class TaskList {
+    private final List<Task> tasks;
```

Comment:
good to see that the plural "tasks" has been used as per the coding standard for naming